python/lsst/afw/image/_exposureSummaryStats.py

```diff
@@ -155,11 +155,9 @@ def _read(bytes):
         if len(droppedFields) > 0:
             droppedFieldString = ", ".join([str(f) for f in droppedFields])
             warnings.warn(
-                (
-                    f"Could not read summary fields [{droppedFieldString}]. "
-                    "Please use a newer stack."
-                ),
+                f"Could not read summary fields [{droppedFieldString}]. Please use a newer stack.",
```

Comment:
Thoughts on how to reword this warning would be welcomed. We get it in obs_base tests because we are reading an old file, so the version of the stack is not all that relevant.

Reply:
In obs_base we should fix the file. Reading it in and writing it out again does fix the warning.

Reply:
Ah, if it's not trying to test backwards compatibility I agree.